frontend/src/css/cat-theme.css

```diff
@@ -19,7 +19,8 @@ body {
 /* Headers */
 
 /* h1: add green line underneath */
-h1.green-divider:after {
+h1.green-divider:after,
+h2.green-divider:after {
```

Comment:
Is this needed still?

Reply:
Good catch! Removed it in commit #https://github.com/code-for-canada/project-thundercat/pull/258/commits/2d2e1d872296040a85c401787289a06e7dc4eaea.